packages/dd-trace/src/lambda/handler.js

```diff
@@ -59,14 +59,31 @@ function crashFlush () {
   }
 }
 
+/**
+ * Extracts the context from the given Lambda handler arguments.
+ *
+ * @param {*} args any amount of arguments
+ * @returns the context, if extraction was succesful.
+ */
+function extractContext (args) {
+  let context = args.length > 1 ? args[1] : undefined
+  if (context === undefined || context.getRemainingTimeInMillis === undefined) {
+    context = args.length > 2 ? args[2] : undefined
+    if (context === undefined || context.getRemainingTimeInMillis === undefined) {
+      throw Error('Could not extract context')
```

Comment:
The tracer shouldn't crash in this case, is there a try/catch block for this?

Reply:
Yup, I was figuring out the best place, currently it's on our instrumentation, couldn't find a better place yet!